localstack/services/s3/s3_listener.py

```diff
@@ -95,6 +95,11 @@
     'Access-Control-Request-Headers', 'Access-Control-Request-Method'
 ]
 
+LOCATION_CONSTRAINT_VALID_REGION = ['af-south-1', 'ap-east-1', 'ap-northeast-1', 'ap-northeast-2', 'ap-northeast-3',
+    'ap-south-1', 'ap-southeast-1', 'ap-southeast-2', 'ca-central-1', 'cn-north-1', 'cn-northwest-1, EU',
```

Comment:
nit: Looks like a typo in string `'cn-northwest-1, EU'`. Also, I think we should probably rather match for `us-east-1` (rather than matching against all the other regions here) - to my knowledge `us-east-1` is literally the only region that is an exception when it comes to `LocationConstraint`.

Reply:
Refractored.